pkg/api/errors/etcd/etcd.go

```diff
@@ -69,6 +69,8 @@ func InterpretUpdateError(err error, qualifiedResource unversioned.GroupResource
 		return errors.NewConflict(qualifiedResource, name, err)
 	case storage.IsUnreachable(err):
 		return errors.NewServerTimeout(qualifiedResource, "update", 2) // TODO: make configurable or handled at a higher level
+	case storage.IsNotFound(err):
```

Comment:
@smarterclayton is there any specific reason we didn't handle not found on etcd update?